recipes/nlohmann_fifo_map/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "nlohmann_fifo_map" %}
+{% set version = "1.0.0" %}
+{% set sha256 = "bb2df8633d6d1a31528191a930d1ee2469ed31fb682fa53a1044962c8f7a1a2e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/nlohmann/fifo_map/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: mkdir "%LIBRARY_INC%\\nlohmann" && move "%SRC_DIR%\\src\\*.hpp" "%LIBRARY_INC%\\nlohmann"  # [win]
+  script: mkdir -p ${PREFIX}/include/nlohmann && mv $SRC_DIR/src/*.hpp --target-dir=${PREFIX}/include/nlohmann  # [unix]
```

I don't know why ``osx`` is complaining, as ``osx`` is not my cup of tea let's try to change that to copy instead of move
```suggestion
  script: mkdir -p ${PREFIX}/include/nlohmann && cp $SRC_DIR/src/*.hpp ${PREFIX}/include/nlohmann  # [unix]
```

---

Yeah, it seems `mv` there does not support `--target_dir`